pytorch_lightning/trainer/trainer.py

```diff
@@ -1598,7 +1604,7 @@ def _distrib_type(self) -> _StrategyType:
         return self._accelerator_connector._distrib_type
 
     @property
-    def _device_type(self) -> DeviceType:
+    def _device_type(self) -> _AcceleratorType:
```

Comment:
will you also rename this?
Since this property is protected and directly tied to this enum, you could rename it here too.

Reply:
Let me do a follow up for this with multiple other changes